docs/organizations/overview.mdx

```diff
@@ -5,26 +5,107 @@ description: Learn about how to create and manage Clerk organizations and their
+
+When a user signs in to your application, they are viewing their personal account and no organization is set as active. Even if they are a member of only one organization, they must explicitly switch to that organization to set it as active.
+
+The easiest way to allow users to switch between their personal account and their organization account is to use Clerk's [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) component. Once a user selects an organization, that organization is set as the active organization.
+
+## Organization management
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations).
+2. In the navigation sidebar, select **Organizations**. Here, you can view and manage all organizations within your application - both those created by you and those create by your users.
+3. Select a specific organization to view its details, members, and settings. Here, you can update the organization's name, slug, and logo. You can also set the organization's [membership limit](#how-many-organizations-can-i-create) and public and private metadata.
+
+### User membership management
+
+Clerk provides a set of prebuilt components to help your users manage their organization memberships:
+
+- [`<CreateOrganization />`](/docs/components/organization/create-organization) - A form for a user to create a new organization.
+- [`<OrganizationProfile />`](/docs/components/organization/organization-profile) - A profile page for the user's currently active organization. Out of the box, this component's **General** tab displays the organization's information and the **Leave organization** option. The **Members** tab shows the organization's members along with their join dates and roles.
+- [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) - A dropdown menu that allows a user to switch between their personal account and their organization account(s).
+- [`<OrganizationList />`](/docs/components/organization/organization-list) - A list of organizations that a user is a member of.
+
+### Admin membership management
+
+As an admin, you can manage organization memberships
+
+- Membership list
+- Domains
+- Invitations (invite form, invitations list)
+- Membership suggestions
+- Membership requests
+
+## Frequently asked questions (FAQ)
+
+### How many organizations can a user belong to?
+
+There is no limit to the number of organizations a user can be a member of.
+
+### How many organizations can my users create?
+
+A user within one of your applications can create up to 100 organizations in that application. This limit is separate from the number of organizations *you* can create across your entire account.
+
+### How many organizations can I create?
```

Comment:
In a dev instance you can create up to 50 organizations. There is no limit for production instances.
After that you will be charged in a different way depending on how many free MAOs your current plan comes with